2019/src/components/Speaker.tsx

```diff
@@ -1,21 +1,60 @@
 import React from "react"
 import styled from "styled-components"
+import { Link } from "gatsby"
+import { useTranslation } from "react-i18next"
+import Image, { FluidObject } from "gatsby-image"
 
-export type Props = {
+export type TalkType = {
+  uuid: string
+  title: string
+  titleJa: string
+  description: string
+  descriptionJa: string
+  spokenLanguage: string
+  slideLanguage: string
+  speakerIDs: string[]
+  date: string
+  startsAt: string
+  endsAt: string
+  room: string
+}
+
+export type SpeakerType = {
+  uuid: string
   name: string
-  photoURL: string
-  talkTitle: string
+  biography: string
+  biographyJa: string
   github: string
   twitter: string
 }
 
+export type AvatarType = {
+  originalName: string
+  originalImg: string
+} & FluidObject
+
+export type Props = {
+  speaker: SpeakerType
+  avatar: AvatarType
+  talk: TalkType
+}
+
+const LinkContainer = styled(Link)`
+  text-decoration: none;
+  color: ${({ theme }) => theme.colors.text};
+`
+const Avatar = styled(Image)`
+  width: 100%;
+  max-width: 273px;
+`
 const Title = styled.h2`
   margin: 0;
   padding: 10px;
   border-bottom: 1px solid ${({ theme }) => theme.colors.border};
   text-align: center;
   font-family: ${({ theme }) => theme.fonts.text};
   font-size: 20px;
+  word-break: break-all;
```

Comment:
Maybe it's good to use `overflow-wrap: break-word;` instead 👀? so that unbreakable word won't break.

| Before | After |
| ------- | ----- |
| <img width="554" alt="image" src="https://user-images.githubusercontent.com/6782666/64934748-e8d57a80-d887-11e9-8ca8-50dafa4c27aa.png"> | <img width="554" alt="image" src="https://user-images.githubusercontent.com/6782666/64934757-f4c13c80-d887-11e9-9917-8ff672dc8eb7.png"> |

Reply:
Hi @CYBAI, thanks for review :+1: